Amplitude/Amplitude.m

```diff
@@ -339,18 +339,16 @@ - (BOOL) migrateEventsDataToDB
     for (id event in events) {
         NSError *error = nil;
         NSData *jsonData = nil;
-        @try {
-            jsonData = [NSJSONSerialization dataWithJSONObject:[AMPUtils makeJSONSerializable:event] options:0 error:&error];
```

Comment:
`dataWithJSONObject` doesn't actually throw any exceptions, so we can get rid of the try / catch block and just check for the error